app/controllers/health_controller.rb

```diff
@@ -32,12 +32,14 @@ def run_mongo_check
     impatient_mongoid_client.close
   end
 
+  # rubocop:disable Style/ClassVars
   def impatient_mongoid_client
```

Comment:
What do you think about making this a class method instead, to avoid these class variables, for example:
```ruby
def self.impatient_mongoid_client
  @impatient_mongoid_client ||= Mongo::Client.new(....)
end
```

Reply:
Sure. How does 00cbfcf4 look? Let me know if you want me to squash these fixups into the original commit.

Reply:
I think it looks really slick, and also thanks for identifying and fixing this issue!